features/admin.actions.v1/constants/actions-constants.ts

```diff
@@ -146,4 +146,13 @@ export class ActionsConstants {
             value: PasswordFormat.PLAIN_TEXT
         }
     ];
+
+    /**
+     * Set of keys used to enable/disable features.
+     * @typeParam FEATURE_DICTIONARY - `Map<string, string>`
+     * @defaultValue
+     */
+    public static readonly FEATURE_DICTIONARY: Map<string, string> = new Map<string, string>()
```

Comment:
can you reuse this instead? https://github.com/wso2/identity-apps/blob/master/features/admin.feature-gate.v1/constants/feature-flag-constants.ts#L35